starters/deno-oak-denodb/deno.json

```diff
@@ -1,4 +1,13 @@
 {
+  "tasks": {
+    "start-db": "docker compose up",
+    "stop-db": "docker compose stop",
+    "start-all": "deno task start-db & deno run --watch --allow-net --allow-env --allow-read ./src/main.ts",
```

Comment:
This is awesome - you can start the Docker container as well as the web server, and both will be automatically shut down when you Ctrl+C.

Reply:
Ok, sure, I added a separate task to start the web server (`start-web`) and `start-db` starts the databases.